yaml/events-cloud-event-proxy-amq-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloud-event-proxy-sidecar
+  namespace: cloud-events
+  labels:
+    app: cloud-event-proxy
+spec:
+  selector:
+    matchLabels:
+      app: cloud-event-proxy
+  template:
+    metadata:
+      labels:
+        app: cloud-event-proxy
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      containers:
+        - name: cloud-event-sidecar
```

Comment:
Do you want to show a sample DU running as main container?

Reply:
Up to you